### PR TITLE
Recime CLI

### PR DESCRIPTION
**Pull request by mehfuzh**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/recime-cli.rb [hunk 1]
```diff
@@ -0,0 +1,92 @@
+
```
Comment (MikeMcQuaid) [inline Formula/recime-cli.rb]: Remove this empty line.

File: Formula/recime-cli.rb [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  version "1.0.5"
+  sha256 "0655a8c84d43c30016f40781cb4043aa1b1167f932e220ddc194c1f22a213acc"
+  head "https://github.com/Recime/recime-cli.git"
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "eb3d2874718ddb753cd89b0307df55e7fa3176f9f6535470dd9011aaf47a64d8" => :sierra
+    sha256 "45d5547354b693df9da372e53c32bea68af4a4a0f699fee3c015dcd6b1b92853" => :el_capitan
+    sha256 "884a3252c287e2db1090389a3d2244abfffef930012f5c194db2614950b7d233" => :yosemite
+  end
```
Comment (MikeMcQuaid) [inline Formula/recime-cli.rb]: Remove this block, we need to generate these.

File: Formula/recime-cli.rb [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", "recime-cli"
+    bin.install "recime-cli"
+
+    # mkdir_p buildpath/"src/github.com/Recime/recime-cli/"
+    # ln_sf buildpath, buildpath/"src/github.com/Recime/recime-cli"
+    # Language::Go.stage_deps resources, buildpath/"src"
+    #
+    # system "go", "build", "-o", bin/"recime-cli", "main.go"
```
Comment (MikeMcQuaid) [inline Formula/recime-cli.rb]: Remove these commented-out lines.

File: Formula/recime-cli.rb [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  end
+
+  test do
+    system bin/"recime-cli", "version"
```
Comment (MikeMcQuaid) [inline Formula/recime-cli.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Formula/recime-cli.rb [hunk 5]
```diff
@@ -0,0 +1,92 @@
+
+require "language/go"
+
+class RecimeCli < Formula
+  desc "Recime command-line tool to create and publish bot."
+  homepage "https://recime.ai"
+  url "https://github.com/Recime/recime-cli/archive/1.0.tar.gz"
+  version "1.0.5"
```
Comment (MikeMcQuaid) [inline Formula/recime-cli.rb]: The version seems to be `1.0` not `1.0.5`.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by MikeMcQuaid** [comment]
> GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
https://bot.brew.sh/job/Homebrew%20Core/11063/version=any_sierra/testReport/junit/brew-test-bot/sierra/audit_recime_cli___new_formula/

Passing on this as a result, sorry! Please consider [creating your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap) instead. Thanks!

**Comment by mehfuzh** [comment]
Does it mean I can't publish this formula to home-brew  with the above exception.

**Comment by MikeMcQuaid** [comment]
Yes, please consider creating your own tap.

**Pull request closed without merging**